src/utils/getImageUrl.js

```diff
@@ -0,0 +1,4 @@
+const imageFallbackPath = 'https://upload.wikimedia.org/wikipedia/commons/8/89/Portrait_Placeholder.png'
+const imagePrefix = 'https://image.tmdb.org/t/p/w500/'
```

Comment:
is this constant the same as `IMAGE_BASE_URL`?

Reply:
no